packages/core/src/feedback/vue/submitFeedback.ts

```diff
@@ -69,16 +69,33 @@ export class FeedbackWebview extends VueWebview {
 
 type FeedbackId = 'AWS Toolkit' | 'CodeWhisperer' | 'Amazon Q' | 'Application Composer'
```

Comment:
Is 'CodeWhisperer' | 'Amazon Q'  supposed to be merged?

Reply:
No, out of scope for now.